extensions/import/src/test/wizard/pages/fileConfigPage.test.ts

```diff
@@ -41,26 +56,212 @@ describe('import extension wizard pages', function () {
+		mockImportModel.object.server.options = testServerConnection.options;
+
+		// expected values for the server dropdown
+		let expectedConnectionValues = [
+			{
+				connection: testActiveConnections[1],
+				displayName: 'testcon2server (testcon2user)',
+				name: 'testConnection2Id'
+			},
+			{
+				connection: testActiveConnections[0],
+				displayName: 'testcon1server (testcon1user)',
+				name: 'testConnection1Id'
+			},
+			{
+				connection: testActiveConnections[2],
+				displayName: 'testcon3server (default)',
+				name: 'testConnection3Id'
+			}
+		];
+
+		//  creating mock database values
+		let databases: string[] = ['testdb1', 'testdb2', 'testdb3'];
+		mockApiWrapper.setup(x => x.listDatabases(TypeMoq.It.isAnyString())).returns(async () => { return databases; });
+		mockImportModel.object.database = 'testdb2';
+
+		// expected values for the database dropdown
+		let expectedDatabaseDropdownValues = [
+			{
+				displayName: 'testdb2',
+				name: 'testdb2'
+			},
+			{
+				displayName: 'testdb1',
+				name: 'testdb1'
+			},
+			{
+				displayName: 'testdb3',
+				name: 'testdb3'
+			}
+		];
+
+		// mock result for the schema query
+		let schemaQueryResult: azdata.SimpleExecuteResult = {
+			rowCount: 3,
+			rows: [
+				[
+					{ displayValue: 'schema1', isNull: false, invariantCultureDisplayValue: 'schema1' }
+				],
+				[
+					{ displayValue: 'schema2', isNull: false, invariantCultureDisplayValue: 'schema2' }
+				],
+				[
+					{ displayValue: 'schema3', isNull: false, invariantCultureDisplayValue: 'schema3' }
+				]
+			],
+			columnInfo: undefined
+		};
+		mockImportModel.object.schema = 'schema2';
+
+		// expected values for the schema dropdown
+		let expectedSchemaValues = [
+			{ displayName: 'schema2', name: 'schema2' }, // This should be the first database as it is active in the extension.
+			{ displayName: 'schema1', name: 'schema1' },
+			{ displayName: 'schema3', name: 'schema3' }
+		];
+
+		let mockQueryProvider = TypeMoq.Mock.ofType(TestQueryProvider);
+		mockApiWrapper.setup(x => x.getProvider(TypeMoq.It.isAny(), TypeMoq.It.isAny())).returns(() => { return mockQueryProvider.object;});
+		mockQueryProvider.setup(x => x.runQueryAndReturn(TypeMoq.It.isAny(), TypeMoq.It.isAny())).returns(async () => { return schemaQueryResult; });
+
+		await new Promise(function (resolve) {
+			page.registerContent(async (view) => {
+				fileConfigPage = new FileConfigPage(mockFlatFileWizard.object, page, mockImportModel.object, view, TypeMoq.It.isAny(), mockApiWrapper.object);
+				pages.set(1, fileConfigPage);
+				await fileConfigPage.start().then(async () => {
+					await fileConfigPage.setupNavigationValidator();
+					resolve();
+				});
+			});
+			wizard.generateScriptButton.hidden = true;
+
+			wizard.pages = [page];
+			wizard.open();
+		});
+
+
+		await fileConfigPage.onPageEnter();
+
+		should.deepEqual(fileConfigPage.serverDropdown.value,expectedConnectionValues[0]);
```

Comment:
Put messages indicating which one failed - if one of these failed we'd have to just guess based on what the expected values were